app/src/main/java/zapsolutions/zap/fragments/SendBSDFragment.java

```diff
@@ -382,7 +390,7 @@ public void onClick(View v) {
                         SendCoinsRequest sendRequest = SendCoinsRequest.newBuilder()
                                 .setAddr(mOnChainAddress)
                                 .setAmount(sendAmount)
-                                .setTargetConf(3)
+                                .setTargetConf(mOnChainFeeView.getFeeTier().getConfirmationBlockTarget())
```

Comment:
@michaelWuensch Actually not a bad idea to also use the calculated fee 😃 
I've tested the usage (see screenshots)

![2019-06-29 23_02_32-Transaction_ 6b7affcc98916a7af3a24590e711fd9b6552da28c25525ff03d49c056c90200e · ](https://user-images.githubusercontent.com/11649986/60389535-20bbf580-9ac3-11e9-9ae5-2b732baf7653.png)
Here you can also see the fix for the wrap of "Gebühr"
![WhatsApp Image 2019-06-29 at 23 05 49](https://user-images.githubusercontent.com/11649986/60389536-21548c00-9ac3-11e9-83ab-845dd207f1d7.jpeg)

Reply:
Awesome, thanks!